lib/transforms/workflow.js

```diff
@@ -0,0 +1,62 @@
+const parseSmartCommit = require('./smart-commit');
+
+function mapStatus({ status, conclusion }) {
+  if (status === 'queued') {
+    return 'in_progress';
```

Comment:
When a workflow runs in GitHub Actions, the app receives the `worflow_run` webhook with the `requested` type and the `queued` status. When that workflow finishes, we then receive another `worflow_run` webhook with the `completed type` and the `success` or `failure` status. Since there's no `in_progress` status (it just moves straight from queued to completed), we're considering `queued` as `in_progress`. 

<img width="219" alt="Screen Shot 2021-06-07 at 11 52 11 am" src="https://user-images.githubusercontent.com/58801956/123719820-d7772a00-d8c5-11eb-8bb0-f633275012cc.png">